File: app/src/main/java/io/tanjundang/study/knowledge/launchmode/FourActivity.java
package io.tanjundang.study.knowledge.launchmode;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import io.tanjundang.study.R;
import io.tanjundang.study.base.BaseActivity;

public class FourActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    protected void initView() {
        setContentView(R.layout.activity_four);
        TextView tvTest = (TextView) findViewById(R.id.tvTest);
        tvTest.setText("当前Activity实例：" + this);
    }

    @Override
    protected void initData() {

    }

    public void SkipToOne(View v) {
        OneActivity.SkipToActivityOne(this, LaunchModeActivity.SINGLETASK);
    }
}

File: app/src/main/java/io/tanjundang/study/knowledge/launchmode/FourActivity.kt
package io.tanjundang.study.knowledge.launchmode

import android.os.Bundle
import android.view.View
import android.widget.TextView

import io.tanjundang.study.R
import io.tanjundang.study.base.BaseActivity

class FourActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun initView() {
        setContentView(R.layout.activity_four)
        val tvTest = findViewById<View>(R.id.tvTest) as TextView
        tvTest.text = "当前Activity实例：$this"
    }

    override fun initData() {

    }

    fun SkipToOne(v: View) {
        OneActivity.SkipToActivityOne(this, LaunchModeActivity.SINGLETASK)
    }
}
